packages/stv/src/index.ts

```diff
@@ -290,15 +288,12 @@ export function redistributeToCandidates(
   organizedVotes: Map<Candidate, CandidateMapItem>,
   candidateSet: Map<Candidate, CandidateMapItem>,
   voteMultiplier: number,
-  votesToRedistribute: number,
-  totalVotesForProportion: number,
 ): void {
   for (const [candidate, vote] of organizedVotes) {
     vote.votes = combineVoteRecords(vote.votes);
     vote.votes.forEach((v) => (v.voteCount *= voteMultiplier));
 
-    const votesToRedistributeForCandidate =
-      votesToRedistribute * (vote.totalVotes / totalVotesForProportion);
+    const votesToRedistributeForCandidate = vote.totalVotes * voteMultiplier;
```

Comment:
This should have been the case from the get-go, see also #3

It is necessary here though, as we want to ensure we use the updated `voteMultiplier`.